sdk/maps/azure-maps-search/README.md

```diff
@@ -0,0 +1,123 @@
+# Azure Maps Search Package client library for Python
+
+This package contains a Python SDK for Azure Maps Services for Search.
+Read more about Azure Maps Services [here](https://docs.microsoft.com/en-us/azure/azure-maps/)
+
+[Source code](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/maps/azure-maps-search) | [Package (Pypi)](https://pypi.org/project/azure-maps-search/) | [API reference documentation](https://docs.microsoft.com/en-us/rest/api/maps/search) | [Product documentation](https://docs.microsoft.com/en-us/azure/azure-maps/)
+
+## _Disclaimer_
+
+_Azure SDK Python packages support for Python 2.7 has ended 01 January 2022. For more information and questions, please refer to https://github.com/Azure/azure-sdk-for-python/issues/20691_
+
+## Getting started
+
+## Prerequisites
+
+- Python 3.6 or later is required to use this package.
+- A deployed Maps Services resource. You can use the [Azure Portal](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource?tabs=windows&pivots=platform-azp) or the [Azure PowerShell](https://docs.microsoft.com/powershell/module/az.communication/new-azcommunicationservice) to set it up.
+
+## Install the package
+
+Install the Azure Maps Service Search SDK.
+
+```bash
+pip install --pre azure-maps-search
+```
+
+## Authenticate the client
+
+In order to interact with the Document Translation feature service, you will need to create an instance of a client.
+An **endpoint** and **credential** are necessary to instantiate the client object.
+
+
+## Looking up the endpoint
+
+You can find the endpoint for your Maps resource using the
+[Azure Portal][azure_portal_get_endpoint].
+
+> Note that the service requires a custom domain endpoint. Follow the instructions in the above link to format your endpoint:
```

Comment:
The instructions about how to get domain endpoint is unclear. Please elaborate more.